src/lib/draw/HoverLayer.svelte

```diff
@@ -18,6 +18,13 @@
     sidebarHover,
   } from "../../stores";
 
+  // Show clickable objects on the map using the cursor
+  $: {
+    $map.getCanvas().style.cursor = $mapHover ? "pointer" : "grab";
```

Comment:
Once other drawing tools also modify the cursor, we'll do this differently -- maybe only running this when we're in attribute or geometry mode.